yt/geometry/_selection_routines/ray_selector.pxi

```diff
@@ -74,6 +75,61 @@ cdef class RaySelector(SelectorObject):
         if total == 0: return None
         return mask.astype("bool")
 
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    @cython.cdivision(True)
+    cdef void visit_grid_cells(self, GridVisitorData *data,
+                              grid_visitor_function *func,
+                              np.uint8_t *cached_mask = NULL) noexcept nogil:
+        # AT PRESENT THIS DOES NOT WORK.
```

Comment:
either this comment is now false and should be removed, or I should stop my review here and wait some more. Which is it ?

Reply:
Neither.  This method is disallowed at a higher-level, by disabling the grid visitor for ray objects, but this code should work and would be a nice breadcrumb for someone else to pick up.